src/microbe_stage/MicrobeStage.cs

```diff
@@ -716,6 +720,24 @@ public void OnFinishTransitioning()
             TutorialEventType.EnteredMicrobeStage, new CallbackEventArgs(HUD.PopupPatchInfo), this);
     }
 
+    /// <summary>
+    ///   Helper function for transition to multicellular
+    ///   Returns array of all microbes of Player's species
+    /// </summary>
+    private Microbe[] GetAllPlayerSpeciesMicrobes()
+    {
+        if (Player == null)
+        {
+            GD.PrintErr("Could not get player species microbes: no Player node");
+            return new Microbe[0];
+        }
+
+        var microbes = rootOfDynamicallySpawned.GetTree().GetNodesInGroup(Constants.AI_TAG_MICROBE).Cast<Microbe>()
+            .ToArray();
```

Comment:
This ToArray is also completely unnecessary, you can just chain LINQ as much as you want (well within reason, we do have a styleguide which doesn't recommend infinite LINQ).